recipes/vulkan-memory-allocator/all/conanfile.py

```diff
@@ -0,0 +1,43 @@
+import os
+
+from conans import ConanFile, tools
+
+
+class VulkanMemoryAllocatorConan(ConanFile):
+    name = "vulkan-memory-allocator"
+    license = "MIT"
+    homepage = "https://github.com/GPUOpen-LibrariesAndSDKs/VulkanMemoryAllocator"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Easy to integrate Vulkan memory allocation library."
+    topics = ("vulkan", "memory-allocator", "graphics")
+    requires = ("vulkan-headers/[>=1.0]")
+    no_copy_source = True
+    # No settings/options are necessary, this is header only
```

Comment:
```suggestion
```

This comment is misleading. Header-only libs may have options or settings, it's orthogonal.

Reply:
This comment comes from a template created by the `conan new` command with the `--header` option. It is also present at the docs [here](https://docs.conan.io/en/latest/howtos/header_only.html).

The comment states that settings/options are not required in case of a header-only library, not that a package can not have them.

I think you are right that the formulation is misleading. I accept the suggested change. Thank you.